packages/utils/upgrade/resources/transforms/5.0.0/upgrade-strapi-version.json.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'node:path';
+import type { JSONTransform } from '../../../src';
```

Comment:
This causes a linting error '"../../../src" is not published  node/no-unpublished-import'